### DISCUSSION TITLE
V190 - AMD GPU

### DISCUSSION THREAD

**Discussion by Raf252** [Rem0o/FanControl.Releases#2511] [General]
Hey!

Not sure if I have missed something but my 7900xt is now showing on Temp Sources with the latest update. Was this intentional :D

**Comment by Rem0o** [comment]
> my 7900xt is now showing on Temp Sources

You mean, it wasn't before and now its present?  Well yeah.

**Reply by Raf252** [reply]
That's right yeah! Well that's great, thanks for everything you do!
